### DISCUSSION TITLE
Why is ShellCheck running on temporary files?

### DISCUSSION THREAD

**Discussion by kasiagauzavlv** [timonwong/vscode-shellcheck#1426] [Q&A] [answered]
The VSCode **Problems** block is polluted with errors like the ones below, and I don't know how to disable this. Any ideas?

![image](https://github.com/vscode-shellcheck/vscode-shellcheck/assets/93767387/f99bd663-1909-4124-9fd7-133dbe63a7d6)

**Comment by kasiagauzavlv** [comment, marked as answer]
It looks like this is an issue with Copilot in VSCode: _Functions in cached chat files are being listed as a reference in the visual studio code editor._

https://github.com/microsoft/vscode-copilot-release/issues/1055

**Reply by felipecrs** [reply]
Makes sense. I'd also recommend you check if Bash IDE isn't contributing these warnings instead of the ShellCheck extension.

Example: https://github.com/vscode-shellcheck/vscode-shellcheck/issues/916#issuecomment-1416605675
